packages/itwin/quantity-formatting/src/components/quantityformat/internal/ScientificType.tsx

```diff
@@ -0,0 +1,94 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import * as React from "react";
+import type { FormatProps } from "@itwin/core-quantity";
+import { parseScientificType, ScientificType as CoreScientificType } from "@itwin/core-quantity";
+import { IconButton, LabeledSelect } from "@itwin/itwinui-react";
+import type { SelectOption } from "@itwin/itwinui-react";
+import { SvgHelpCircularHollow } from "@itwin/itwinui-icons-react";
+import { useTranslation } from "../../../useTranslation.js";
+import "../FormatPanel.scss";
+
+/** Properties of [[ScientificType]] component.
+ * @internal
+ */
+export interface ScientificTypeProps {
+  formatProps: FormatProps;
+  onChange?: (format: FormatProps) => void;
```

Comment:
Why is `onChange` optional?

Reply:
that's an outdated thing from the old component, I've changed it to be required